docs/prod-customize.md

```diff
@@ -9,6 +9,16 @@ by a system administrator.
 
 [realm-admin-docs]: https://zulipchat.com/help/getting-your-organization-started-with-zulip
 
+
+## Authentication Backends
+
+`AUTHENTICATION_BACKENDS` is a list of enabled authentication mechanisms. You'll
+need to enable at least one authentication mechanism by uncommenting its
+corresponding line, and then also do any additional configuration required for
```

Comment:
we should probably have this say that by default, just email is enabled, rather than that you definitely need to uncomment one.